.github/workflows/build_docs.yml

```diff
@@ -1,12 +1,15 @@
-name: documentation
+# build the documentation and upload the built artifact
+name: build documentation
 
 on:
   workflow_call:
   workflow_dispatch:
+  # this will run a build when a review is submitted to make sure the PR does not break docs before merging
+  pull_request_review:
```

Comment:
whats the diff between on "pull_request" and "pull_request_review"?

Reply:
pull_request_review triggers on PR review activity. Right now I have it set for when a review is submitted or edited. 
pull_request triggers on any push once a pull request is open.

I set it to pull_request_review to avoid running it on every push, and only once it ready to be merged.

Reply:
FYI, this triggers every time something happens in the review (review comment, etc). But if you then commit after the tests don't get re-run (for better or worse). Maybe we should also add a manual trigger.

Reply:
The build_docs workflow already has a [manual_dispatch](https://github.com/FAIR-Chem/fairchem/blob/c840e4716d66eef0bf546b630c6b51150c5d5140/.github/workflows/build_docs.yml#L6) trigger, is this what you mean?

I can change the pull_request_review to only run on submitted. The purpose here precisely so docs builds dont get triggered on subsequent pushes, because they take too long.

I think the two triggers we have are sufficient:
- on pull_request_review submitted, so that new we are alerted if a PR breaks the docs right before merging.
- manual_dispatch for the few cases, like this one, when the docs are being updated. The developer is responsible to run them when necessary as they develop.

Thoughts?